server-toss/tests/load-test/README.md

```
@@ -1,37 +1,331 @@
+매 실행마다
+
+```
+results/<runId>/
+```
+
+가 생성됩니다.
 
-```sh
-# 기본 1,000명
-pnpm -F server-toss seed:drawing:large
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)